apps/web/package.json

```
@@ -52,8 +51,7 @@
     "react-editor-js": "^1.9.0",
     "react-hook-form": "^7.2.3",
     "react-query": "^3.5.16",
-    "react-router": "5.2.0",
-    "react-router-dom": "5.2.0",
+    "react-router-dom": "6",
```

```suggestion
    "react-router-dom": "6.2.2",
```
Maybe pin to a specific version here? Does npm handle this as `^6.2.2`? not familiar with this type of syntax

---

let me fix this